src/components/garden_admin_card.tsx

```diff
@@ -1,83 +1,146 @@
 import useGardens from 'hooks/useGardens'
+import { Dispatch, SetStateAction, useState } from 'react'
 import { Link as RouterLink } from 'react-router-dom'
 import { Garden } from 'types/backend-types'
-import { TableState } from 'types/custom-types'
+import { SnackbarState } from 'types/custom-types'
 
 interface GardenAdminCardProps {
   garden: Garden
+  setRequestStatus: Dispatch<SetStateAction<SnackbarState | undefined>>
 }
 
-const GardenAdminCard = ({ garden }: GardenAdminCardProps) => {
+const GardenAdminCard = ({
+  garden,
+  setRequestStatus,
+}: GardenAdminCardProps) => {
+  const [open, setOpen] = useState(false)
+
   const { deleteGarden } = useGardens()
-  function getTableData() {
-    return [
-      ['Status', garden.status],
-      ['Namespaces', garden.namespaces.length],
-      ['Systems', garden.systems.length],
-    ]
-  }
 
-  const state: TableState = {
-    formatData: getTableData,
-    tableHeads: [],
-    includePageNav: false,
-    disableSearch: true,
+  const style = {
+    position: 'absolute' as const,
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    bgcolor: 'background.paper',
+    boxShadow: 24,
+    p: 4,
+    height: 'auto',
+    overflow: 'auto',
   }
 
-  function getDeleteButton(connection_type: string) {
-    if (connection_type !== 'LOCAL') {
-      return (
-        <Button
-          onClick={() => deleteGarden(garden.name)}
-          variant="contained"
-          color="secondary"
-        >
-          Delete {garden.name}
-        </Button>
-      )
-    }
-  }
+  const handleDeleteClick = () => {
+    deleteGarden(garden.name)
+      .then(() => {
+        setRequestStatus({
+          severity: 'success',
+          message: `Garden: ${garden.name} successful removed`,
+          showSeverity: false,
+        })
+        setOpen(false)
+      })
+      .catch((error) => {
+        console.error('ERROR', error)
 
-  function localOrRemote(connection_type: string) {
-    if (connection_type === 'LOCAL') {
-      return '(LOCAL)'
-    } else {
-      return '(REMOTE)'
-    }
+        if (error.response) {
+          setRequestStatus({
+            severity: 'error',
+            message: `${error.response.data.message}`,
+            doNotAutoDismiss: true,
+          })
+        } else {
+          setRequestStatus({
+            severity: 'error',
+            message: `${error}`,
+          })
+        }
+      })
   }
 
   return (
     <Card sx={{ minWidth: 275 }}>
       <AppBar color="inherit" position="static">
         <Toolbar>
           <Typography variant="h6" color="inherit">
-            {garden.name} {localOrRemote(garden.connection_type)}
+            {garden.name}{' '}
+            {garden.connection_type === 'LOCAL' ? '(LOCAL)' : '(REMOTE)'}
           </Typography>
         </Toolbar>
       </AppBar>
       <CardContent>
-        <Table parentState={state} />
+        <Grid container>
+          <Grid>Status: </Grid>
```

Comment:
I love the colored status indicators! Can we possibly get just a little more whitespace between the colon after the word *Status* and the colored indicator? They look kind of smushed together:

<img width="172" alt="image" src="https://user-images.githubusercontent.com/73488256/197785570-87bcb81d-037e-4b9d-8c23-9d105e61777a.png">